docs/source/administrator/upgrading.md

```diff
@@ -100,19 +76,27 @@ will be performed automatically when you do a `helm upgrade`.
 4. Do a [`helm upgrade`](helm-upgrade-command). This should perform the database upgrade needed.
 5. Remove the lines added in step 3, and do another [`helm upgrade`](helm-upgrade-command).
 
-### Custom Docker Images: JupyterHub version match
+## Custom Docker Images: JupyterHub version match
 
 If you are using a custom built image, make sure that the version of the
-JupyterHub package installed in it is now 0.8.1. It needs to be 0.8.1 for it to work with
-v0.6 of the helm chart.
+JupyterHub package installed in it matches the major version of JupyterHub, current 2.\*.
 
 For example, if you are using `pip` to install JupyterHub in your custom Docker Image,
 you would use:
 
 ```Dockerfile
-RUN pip install --no-cache-dir jupyterhub==0.8.1
+RUN pip install --no-cache-dir jupyterhub==2.3.1
 ```
 
+If you are using conda or mamba:
+
+```Dockerfile
+RUN conda install -y jupyterhub-base=2.3.1
```

Comment:
Let's point to conda-forge, I'm not sure if we are available quickly in the default channel.

```suggestion
RUN conda install --channel=conda-forge -y jupyterhub-base=2.3.1
```